docker-compose.yml

```diff
@@ -146,6 +146,7 @@ services:
       PROXY_PRIMARY_URL: ${PROXY_PRIMARY_URL:-local-dev.dataland.com}
       INVITATION_REQUEST_RECEIVERS: ${INVITATION_REQUEST_RECEIVERS}
       INVITATION_REQUEST_CC: ${INVITATION_REQUEST_CC}
+      BACKEND_MIGRATION_BASELINE_VERSION: ${BACKEND_MIGRATION_BASELINE_VERSION:-1}
```

Comment:
The :-1 sets a default value of 1? Does that mean that we no longer need the default environment variable? If we still need it than I guess that the :-1 doesn't actually have the desired effect and is not needed.

Reply:
We use the :-1 for the local environment, I don't want to enforce everyone to set this env.

Reply:
On dev PCs